content/ml/OWNERS

```diff
@@ -0,0 +1,12 @@
+approvers:
+  - radhikapc
+  - sreedaum
+  - leodido
+  - fntlnz
+  - Rajakavitha1
```

Comment:
```suggestion
```

Removing root level maintainers (they get inherited from the root OWNERS file). Leaving here only the maintainers responsible for Malayam :)